homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -220,11 +226,11 @@ async def async_alarm_arm_night(self, code: str | None = None) -> None:
             ) from error
         await self.coordinator.async_request_refresh()
 
-    def _arm_night(self, code=None):
+    def _arm_night(self) -> None:
         """Arm night synchronous."""
         ArmingHelper(self._partition).arm_stay_night()
 
-    async def async_alarm_arm_home_instant(self, code: str | None = None) -> None:
+    async def async_alarm_arm_home_instant(self) -> None:
```

Comment:
Feels strange this would not require any code then?

Reply:
The code is not really required by the underlying total_connect_client API calls, and this `arm_home_instant` is an integration specific function not supported by the base `alarm_control_panel` user interface, so a PIN will never be entered.

Reply:
Would have been better to have this "refactoring" in a separate PR as it's hard to follow the usage of the code now.